third-party/mage-hand-press/archetypes/monk-fourelementsremastered.xml

```diff
@@ -0,0 +1,780 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<elements>
+  <info>
+    <name>Way of the Four Elements (Remastered)</name>
+    <description>Way of the Four Elements (Remastered) archetype.</description>
+    <author url="https://drive.google.com/file/d/0B1pdYIcfHauwNDM2My1XeWFYSDA/view?usp=sharing">SpiketailDrake and IrishBandit</author>
+    <update version="0.1">
+      <file name="monk-fourelementsremastered.xml" url="https://raw.githubusercontent.com/AnkurD/elements/5833718ed6a606bd8b4aa614fa13c4b46f5efa0e/third-party/mage-hand-press/archetypes/monk-fourelementsremastered.xml" />
+    </update>
+  </info>
+  <!-- Way of the Four Elements (Remastered) -->
+  <element name="Way of the Four Elements (Remastered)" type="Archetype" source="SpiketailDrake and IrishBandit" id="ID_RDDT_ARCHETYPE_MONK_WAY_OF_THE_FOUR_ELEMENTS_REMASTERED">
+    <supports>Monastic Tradition</supports>
+    <description>
+      <p>You follow a monastic tradition that teaches you to harness the elements. When you focus your ki, you can align yourself with the forces of creation and bend the four elements to your will, using them as an extension of your body. Some members of this tradition dedicate themselves to a single element, but others weave the elements together.</p>
+      <p class="indent">Many monks of this tradition tattoo their bodies with representations of their ki powers, commonly imagined as coiling dragons, but also as phoenixes, fish, plants, mountains, and cresting waves.</p>
+      <div element="ID_WOTC_PHB_ARCHETYPE_FEATURE_WAY_OF_THE_FOUR_ELEMENTS_DISCIPLE_OF_THE_ELEMENTS" />
+    </description>
+    <sheet>
+      <description>You follow a monastic tradition that teaches you to harness the elements.</description>
+    </sheet>
+    <rules>
+      <grant type="Archetype Feature" name="ID_WOTC_PHB_ARCHETYPE_FEATURE_WAY_OF_THE_FOUR_ELEMENTS_DISCIPLE_OF_THE_ELEMENTS_REVISED" level="3" />

```

Comment:
You're referencing the WOTC Feature here (with a revised keyword), but below, on line 26, you've defined a different ID for the Feature.

I encountered no other mechanical issues when testing it and all the grants worked as they appeared they should.

Reply:
Thanks for noticing that. I tried to correct that and update some description ordering.